extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -118,6 +118,9 @@ class AmpSocialShare extends AMP.BaseElement {
     if (!element.hasAttribute('tabindex')) {
       element.setAttribute('tabindex', '0');
     }
+    if (!element.getAttribute('aria-label')) {
+      element.setAttribute('aria-label', `Share by ${typeAttr}`);
```

Comment:
Another aspect to consider here, (not in scope though), is localization of the default value.

Reply:
Great recommendation, thanks Kris.